.gitignore

```diff
@@ -71,3 +71,4 @@ ENV/
 /tmp/
 *.sw[po]
 **/tags
+/.idea/
```

Comment:
You can add local git ignores to ``$GIT_DIR/info/exclude``, see https://git-scm.com/docs/gitignore